src/components/SearchBox/onSelectedFactory.ts

```diff
@@ -34,7 +34,7 @@ const getSkeleton = (option) => {
   };
 };
 
-export const onHighlightFactory = (setPreview) => (e, location) => {
+export const onHighlightFactory = (setPreview, location) => {
```

Comment:
This is not a factory any longer. Factory means it creates a "onHighlight" function in this context. Another correct name for such function is High order function.

The proper name now would be just `onHighlight`